tensorflow/lite/tools/accuracy/ilsvrc/README.md

```diff
@@ -56,6 +56,10 @@ and the following optional parameters:
     Optionally, the computed accuracies can be output to a file as a
     string-serialized instance of tflite::evaluation::TopkAccuracyEvalMetrics.
 
+*   `num_ranks`: `int` (default=10) \
+    The number of top accuracy counts, if 5, top-1 to top-5 rank result will
```

Comment:
Please format as: "The number of top-K accuracies to return. For example, if num_ranks=5, top-1 to top-10 accuracy fractions are returned."

Reply:
fixed.